### PR TITLE
Pricing doc

### PR DESCRIPTION
**Pull request by franzpgarcia**
Updated pricing doc + added changes apis

### CODE REVIEW

File: source/includes/administration/_pricings.md [hunk 1]
```diff
@@ -20,458 +20,761 @@ curl "https://cloudmc_endpoint/rest/pricings" \
...
+          "unitPrice": {
+            "CAD": 13
+          },
+          "product": {
+            "id": "dd3fcab9-5b31-4f08-9b50-ed3326bccfb4",
+          },
+          "cogs": {
+            "CAD": 10
+          },
+          "deprecated": false,
```
Comment (GuillaumeVialle) [inline source/includes/administration/_pricings.md]: I don't think we have to specify that upon creation.

File: source/includes/administration/_pricings.md [hunk 2]
```diff
@@ -20,458 +20,761 @@ curl "https://cloudmc_endpoint/rest/pricings" \
...
+
+Priced product
+
+Required | &nbsp;
+---- | -----------
+`id`<br/>*UUID* | The id of the priced product
+`product`<br/>*Product* | The product being priced
+`cogs`<br/>*Map* | Map from currency code to cogs
+`unitPrice`<br/>*Map* | Map from currency code to unit price
+`deprecated`<br/>*Boolean* | True if priced product is deprecated
```
Comment (GuillaumeVialle) [inline source/includes/administration/_pricings.md]: seems odd to create a depreciated pricing?

File: source/includes/administration/_pricings.md [hunk 3]
```diff
@@ -20,458 +20,761 @@ curl "https://cloudmc_endpoint/rest/pricings" \
...
 }
 ```
 
-> The above command return JSON structured like this:
+Required | &nbsp;
+---- | -----------
+`id`<br/>*UUID* | The id of the pricing
+`name`<br/>*Map* | A map from language to name value for that language
+`description`<br/>*Map* | A map from language to description value for that language
+`supportedCurrencies`<br/>*List* | A list of currencies supported by the pricing
```
Comment (GuillaumeVialle) [inline source/includes/administration/_pricings.md]: just the name and description is required here

File: source/includes/administration/_pricings.md [hunk 4]
```diff
@@ -20,458 +20,761 @@ curl "https://cloudmc_endpoint/rest/pricings" \
...
+        {
+          "productId": "dd3fcab9-5b31-4f08-9b50-ed3326bccfb4",
+          "field": "unitPrice",
+          "currency": "CAD",
+          "value": 13
+        }
+    ],
+    "effectiveDate": "2020-09-02T12:00:00Z",
+    "pricingChangeType": "MODIFY_PRODUCTS"
+,
```
Comment (GuillaumeVialle) [inline source/includes/administration/_pricings.md]: that comma should be a line above

File: source/includes/administration/_pricings.md [hunk 5]
```diff
@@ -20,458 +20,761 @@ curl "https://cloudmc_endpoint/rest/pricings" \
...
+
+Update a pricing change to a pricing
+
+```shell
+# Update a pricing change
+curl -X PUT "https://cloudmc_endpoint/v2/pricings/73eb0d75-03b0-44e2-9cbf-9ad25dff5da5/changes/920d7ee9-8771-4485-a0f2-2e122db83c32" \
+   -H "MC-Api-Key: your_api_key"
+```
+
+See Add Pricing Change for body. You cannot change the `pricingChangType`
```
Comment (GuillaumeVialle) [inline source/includes/administration/_pricings.md]: pricingChangeType, typo here

File: source/includes/administration/_pricings.md [hunk 6]
```diff
@@ -20,458 +20,761 @@ curl "https://cloudmc_endpoint/rest/pricings" \
...
+
+
+<!-------------------- REMOVE PRICING CHANGES -------------------->
+### Remove pricing change
+
+`DELETE /pricings/:id/changes/:change_id`
+
+Remove a pricing change of a pricing
```
Comment (GuillaumeVialle) [inline source/includes/administration/_pricings.md]: future*

File: source/includes/administration/_pricings.md [hunk 7]
```diff
@@ -20,458 +20,761 @@ curl "https://cloudmc_endpoint/rest/pricings" \
...
+`currency`<br/>*string* | Currency of value to change.
+`value`<br/>*double* | New price for specifified field.
+
+
+<!-------------------- UPDATE PRICING CHANGES -------------------->
+### Update pricing change
+
+`PUT /pricings/:id/changes/:change_id`
+
+Update a pricing change to a pricing
```
Comment (GuillaumeVialle) [inline source/includes/administration/_pricings.md]: future*

File: source/includes/administration/_pricings.md (hunk 1 above)
Comment (franzpgarcia) [reply source/includes/administration/_pricings.md]: done

File: source/includes/administration/_pricings.md (hunk 2 above)
Comment (franzpgarcia) [reply source/includes/administration/_pricings.md]: removed

File: source/includes/administration/_pricings.md (hunk 3 above)
Comment (franzpgarcia) [reply source/includes/administration/_pricings.md]: oups you're right. done

File: source/includes/administration/_pricings.md (hunk 4 above)
Comment (franzpgarcia) [reply source/includes/administration/_pricings.md]: fixed

**Review by evanluc — APPROVED** [review]

**Review by GuillaumeVialle — APPROVED** [review]
Thanks for the changes

**Pull request merged**